frontend/javascripts/types/api_flow_types.ts

```diff
@@ -537,15 +537,36 @@ export type APIAnnotationWithTask = APIAnnotationBase & {
 export type APITaskWithAnnotation = APITask & {
   readonly annotation: APIAnnotation;
 };
-export type APITimeTracking = {
-  time: string;
-  timestamp: number;
+export type APITimeTrackingPerAnnotation = {
   annotation: string;
-  _id: string;
-  task_id: string | undefined;
-  project_name: string | undefined;
-  tasktype_id: string | undefined;
-  tasktype_summary: string | undefined;
+  task: string | undefined;
+  projectName: string | undefined;
+  timeMillis: number;
+  annotationLayerStats: Array<TracingStats>;
+};
+export type APITimeTrackingPerUser = {
+  user: {
+    id: string;
+    firstName: string;
+    lastName: string;
+    email: string;
```

Comment:
can you define this type as an extension of `APIUserCompact` (similar to `APIUserBase`) in api_flow_types.ts?